src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -1659,20 +1666,13 @@ public void testCachedFile() throws Exception {
         checker.destroy();
     }
 
-    @SuppressForbidden
     @Test
     public void testUnmappableCharacters() throws Exception {
         final String[] expected = {
-            "4: " + getCheckMessage(LineLengthCheck.class, MSG_KEY, 75, 238),
+            "9: " + getCheckMessage(LineLengthCheck.class, MSG_KEY, 80, 100),
         };
 
-        final DefaultConfiguration checkConfig = createModuleConfig(LineLengthCheck.class);
-        checkConfig.addProperty("max", "75");
-
-        final DefaultConfiguration checkerConfig = createRootConfig(checkConfig);
-        checkerConfig.addProperty("charset", "IBM1098");
```

Comment:
Please use xml formatted config in Input file, and set this property to Checker.

Reply:
Good evening @romani , I tried to write the properties in xml in Input file but it gives me "Checkstyle Config comment not specified properly". This is my config comment:

`/*xml
<module name="Checker">
  <property name="charset" value="IBM1098"/>
  <module name="LineLength">
    <property name="max" value="75"/>
  </module>
</module>

*/`

Could you help me please? Did I write the xml wrong?

Reply:
GoodMorning @romani, 
I debugged it:

[
![Screenshot 2024-02-17 115924](https://github.com/checkstyle/checkstyle/assets/99334169/6d7a9262-7da8-4d13-81f6-42ca8f43a579)
](url)

The problem was that the "getFullyQualifiedClassName" method was unable to return the "FullyQualifiedClassName" for "LineLength." So, since the method expects it, I passed the FullyQualifiedName directly in the XML configuration. This change makes the test pass. If passing the FullyQualifiedName directly doesn't seem elegant as a solution, I can add another entry to the Map "moduleMappings" so that I can obtain the FullyQualifiedName for "LineLength" within the "getFullyQualifiedClassName" method instead of passing it directly.

Reply:
nice!!!